lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -158,5 +158,16 @@ def api_token
     def api_url
       @api_url ||= MiqRegion.my_region.remote_ws_url
     end
+
+    def miq_request_task_url
+      result = {}
+      if @workspace.root['vmdb_object_type']
+        vmdb_obj = @workspace.root[@workspace.root['vmdb_object_type']]
+        if vmdb_obj.kind_of?(MiqAeMethodService::MiqAeServiceMiqRequestTask)
+          result['task_href_slug'] = "#{vmdb_obj.miq_request.href_slug}/#{vmdb_obj.href_slug}"
```

Comment:
The key name `task_href_slug` is not consistent with the other `href_slug` keys we pass.  Should this be `miq_request_task` or `request_task`?

cc @syncrou